src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -274,17 +274,17 @@
         </td>
       </tr>
 
-      {% if events is defined %}
+      {% if events is defined and events is not empty %}
```

Comment:
Isn't `events is not empty` enough? A http://www.mitchellbosecke.com/pebble/documentation/test/empty says that it handle nulls.

Reply:
done